Bottom: 2adb3c3405ef1b15aa3d8d31a836fcb5fe5baab2
Top:    5d156e7750c68c5ec7bab8c8982fdd2d8e2ef90b
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-01 16:14:04 +0800

tuntap: switch to use XDP_PACKET_HEADROOM

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index f0aa243..4506da3 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -113,7 +113,6 @@ do {								\
 } while (0)
 #endif
 
-#define TUN_HEADROOM 256
 #define TUN_RX_PAD (NET_IP_ALIGN + NET_SKB_PAD)
 
 /* TUN device flags */
@@ -1653,7 +1652,7 @@ static struct sk_buff *tun_build_skb(struct tun_struct *tun,
 	rcu_read_lock();
 	xdp_prog = rcu_dereference(tun->xdp_prog);
 	if (xdp_prog)
-		pad += TUN_HEADROOM;
+		pad += XDP_PACKET_HEADROOM;
 	buflen += SKB_DATA_ALIGN(len + pad);
 	rcu_read_unlock();
